fact/src/host_info.rs

```diff
@@ -64,3 +70,35 @@ pub fn get_username(uid: u32) -> &'static str {
         None => "",
     }
 }
+
+// get_mnt_namespace
+//
+// Returns a mount namespace of the host. Since we're running as a privileged
+// process, it's equivalent to our mount namespace, thus extract it as an inode
+// of /proc/self/ns/mnt . We should get the same result if we try to use
+// /proc/1/ns/mnt .
+pub fn get_mnt_namespace() -> &'static u64 {
```

Comment:
Since `u64` implements `Copy` I don't think we need a `&'static`, we can just make a copy when we call the function.
```suggestion
pub fn get_mnt_namespace() -> u64 {
```

This may also require line 103 to be changed to something like `*HOST_MNT_NAMESPACE` I think.